### DISCUSSION TITLE
React native support?

### DISCUSSION THREAD

**Discussion by dmahajan980** [vanilla-extract-css/vanilla-extract#575] [Q&A]
I liked the concept of vanilla extract and was wondering if it could support React Native in the future? Support native might be a lot of work, though it might be great to know if the team has any similar plans.

**Comment by gabrielgrover** [comment]
Also very interested in this

**Comment by ryan0x44** [comment]
Also interested in this. Seems Reddit users are too https://www.reddit.com/r/reactnative/comments/q8ijg9/decent_cssinjs_libraries_for_react_native/

If folks want to use the same React Native (RN) components across native and web they can now use React Native Web (RNW) - docs for styling in RNW here: https://necolas.github.io/react-native-web/docs/styling/

**Comment by marklawlor** [comment]
I created a proof of concept, see the example folder for a working demo.

Still experimental, but variables & media queries should work

https://github.com/marklawlor/neapolitan-extract
